spec/index.html

```diff
@@ -167,57 +167,71 @@ <h3>Resources and Statements</h3>
       exists, without explicitly naming it.</p>
   </section>
 
-  <section id="section-quoted-triples">
-    <h3>Quoted Triples</h3>
+  <section id="section-graph-terms">
+    <h3>Graph Terms</h3>
 
-    <p>A <a>quoted triple</a> is an <a>RDF term</a> with the components of
-      an <a>RDF triple</a>, which can be used as the <a>subject</a> or <a>object</a>
+    <p>A <a>graph term</a> is an <a>RDF term</a> with the components of
+      an <a>RDF graph</a>, which can be used as the <a>subject</a> or <a>object</a>
       of another triple.</p>
```

Comment:
While this sentence made sense when it was talking about quoted triples, it does not make sense anymore now that it is talking about graph terms. I suggest to change it as follows.
```suggestion
    <p>A <a>graph term</a> is an <a>RDF graph</a>
      used as the <a>subject</a> or <a>object</a>
      of a triple.</p>
```